samples/openthread/ncp/README.rst

```diff
@@ -0,0 +1,88 @@
+.. _ot_ncp_sample:
+
+Thread: NCP sample
```

Comment:
```suggestion
Thread: NCP
```
`sample` should never be part of the sample name.

Reply:
Done